packages/ember-data/lib/system/model/model.js

```diff
@@ -1061,15 +683,9 @@ var Model = Ember.Object.extend(Ember.Evented, {
     successfully or rejected if the adapter returns with an error.
   */
   save: function() {
-    var promiseLabel = "DS: Model#save " + this;
-    var resolver = Ember.RSVP.defer(promiseLabel);
-
-    this.store.scheduleSave(this, resolver);
-    this._inFlightAttributes = this._attributes;
-    this._attributes = Ember.create(null);
-
-    return PromiseObject.create({
-      promise: resolver.promise
+    var model = this;
+    return this.reference.save().then(function() {
+      return model;
     });
```

Comment:
Previously a `PromiseObject` has been returned here, where now the promise itself is returned. Is this intended?